projects/1.16-fabric/assets/ringsofascension/zh_cn.json

```
@@ -0,0 +1,65 @@
+{
+    "itemGroup.ringsofascension.rings": "提升戒指",
+  
+    "item.ringsofascension.ring_fire_resistance": "抗火戒指",
+    "item.ringsofascension.ring_invisibility": "隐身戒指",
+    "item.ringsofascension.ring_regeneration": "再生戒指",
+    "item.ringsofascension.ring_slow_falling": "羽落戒指",

```

```suggestion
    "item.ringsofascension.ring_slow_falling": "缓降戒指",
```
原文slow falling在原版中是`缓降`